### ISSUE TITLE
RenPy 7.1.0 fails to build from release archive

### ISSUE THREAD

**Issue by jbeich** [renpy/renpy#1563]
4a9efc0cab7d and fe9af97b34cd switched to bundled fribidi but neither [sdk](https://renpy.org/dl/7.1.0/renpy-7.1.0-sdk.tar.bz2) nor [source](https://renpy.org/dl/7.1.0/renpy-7.1.0-source.tar.bz2) contain `module/fribidi-src/`. Excerpt from [build log](https://ptpb.pw/UG0J):
```c
gen/_renpybidi.c:564:10: fatal error: 'fribidi.h' file not found
#include "fribidi.h"
         ^
```

**Comment by jbeich** [comment]
Downstream I've [switched back](https://github.com/freebsd/freebsd-ports/blob/cba0358c6dae/games/renpy/files/patch-system-fribidi) to system fribidi as a workaround. Extracting bundled version from a different source is a bit complicated while converting to GitHub automatic archives is blocked by tags being out of sync with actual releases.

**Comment by hasufell** [comment]
Yes, the archive is broken. In addition, the bundling of fribidi is not very nice. Why can't we use the system version?

**Comment by renpytom** [comment]
The reason we bundle fribidi is because, at least until recently, it had as a dependency glib, which I didn't want to include with Ren'Py or detect and support in its build scripts. For something as small and well defined as fribidi, it makes sense to just include the relevant bits rather than subjecting users to large downloads and dependency hell.

**Comment by hasufell** [comment]
> For something as small and well defined as fribidi, it makes sense to just include the relevant bits rather than subjecting users to large downloads and dependency hell.

How you bundle and build your binaries is something different from how you structure your source code and your build system. Not supporting system libraries is always something that decreased linux distro coverage. In addition, it's hard to get right, as you can see.

**Comment by renpytom** [comment]
This should have been resolved by 7.1.1.
